addons/hr/__manifest__.py

```diff
@@ -20,7 +20,7 @@
         'mail',
         'resource',
         'web',
-        'mail_bot',
+        'mail_bot_hr'
```

Comment:
Nope, I would instead just have mail_bot_hr bridge between mail_bot and hr . Don't add explicit dependencies, unless there is a reason ?